libs/langchain-azure-cosmosdb/src/caches.ts

```diff
@@ -0,0 +1,172 @@
+import {
+  BaseCache,
+  deserializeStoredGeneration,
+  getCacheKey,
+  serializeGeneration,
+} from "@langchain/core/caches";
+import { Generation } from "@langchain/core/outputs";
+import { Document } from "@langchain/core/documents";
+import { EmbeddingsInterface } from "@langchain/core/embeddings";
+import { CosmosClient, CosmosClientOptions } from "@azure/cosmos";
+import { DefaultAzureCredential } from "@azure/identity";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import {
+  AzureCosmosDBNoSQLConfig,
+  AzureCosmosDBNoSQLVectorStore,
+} from "./azure_cosmosdb_nosql.js";
+
+const USER_AGENT_SUFFIX = "LangChain-CDBNoSQL-SemanticCache-JavaScript";
+const DEFAULT_DATABASE_NAME = "CosmosNoSqlCacheDB";
+const DEFAULT_CONTAINER_NAME = "CosmosNoSqlCacheContainer";
```

Comment:
```suggestion
const DEFAULT_CONTAINER_NAME = "semanticCacheContainer";
```
Aligning on other integrations